.github/workflows/ci.yml

```diff
@@ -7,15 +7,19 @@ name: CI
 jobs:
   test:
     runs-on: ubuntu-latest
+    container: alpine:3.15
     steps:
-    - name: go
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.16.x
-    - name: checkout
-      uses: actions/checkout@v2
-    - name: test
-      run: go test ./...
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: install-go
+        run: apk add go
+      - name: build-ctags
+        run: |
+          export SOURCEGRAPH_COMMIT=70028a0bde4bbffe1919635b3d1fee99c2e28625
```

Comment:
Can we put this in a file or something so we don't have to update multiple places?